coral-collection/src/scripts/SelectableCollection.js

```diff
@@ -112,16 +117,21 @@ class SelectableCollection extends Collection {
    @protected
    */
   _getNextSelectable(item) {
-    let sibling = item.nextElementSibling;
+    const items = this.getAll();
+    let index = items.indexOf(item);
+    let sibling = index < items.length - 1 ? items[index + 1] : null;
+
     while (sibling) {
       if (sibling.matches(this._selectableItemSelector)) {
         break;
       }
       else {
-        sibling = sibling.nextElementSibling;
+        index++;
+        sibling = index < items.length - 1 ? items[index + 1] : null;
       }
     }
-    
+  
+    // in case the item is not specified, or it is not inside the collection, we need to return the first selectable
     return sibling || item;
```

Comment:
1. The comment says : in case the item is not specified, or it is not inside the collection, we need to return the first selectable
But we return item not first selectable.
2. In case the (index == items.length - 1) last element - we return item itself. Is this expected?

Reply:
1. seems like I copied and pasted the comment.
2. I think this is the expected behavior, if we are on the last element we shouldn't navigate further. Returning the item itself is safer than null.